src/content-scripts/SelectionRect.js

```diff
@@ -123,8 +123,17 @@ class Rect {
  */
 class DocRect extends Rect {
 	constructor(doc) {
-		let docElem = document.documentElement;
-		super(0, 0, docElem.scrollHeight, docElem.scrollWidth);
+    let docElem = document.documentElement;
+    let docHeight = docElem.scrollHeight;
+    // To handle some websites (#288)
+    if(docHeight == 0) {
+      docHeight = Math.max(
+        document.body.scrollHeight, docElem.scrollHeight,
+        document.body.offsetHeight, docElem.offsetHeight,
+        document.body.clientHeight, docElem.clientHeight
+      );
+    }
+    super(0, 0, docHeight, docElem.scrollWidth);
```

Comment:
Technically if we're compensating for height peculiarities, the width should be accommodated as well.